pkg/chrootarchive/diff_unix.go

```diff
@@ -49,6 +51,10 @@ func applyLayer() {
 		fatal(err)
 	}
 
+	if inUserns {
+		options.InUserNS = true
```

Comment:
(even still, couldn't this just be `options.InUserNS = inUserns`?  or does setting it regardless like that have other consequences?)